caffe2/ideep/operators/operator_fallback_ideep.h

```diff
@@ -109,13 +109,23 @@ class IDEEPFallbackOp final : public IDEEPOperator {
           "output type who needs copying.");
       const auto& src = local_output_blobs_[i]->template Get<TensorCPU>();
 
+      auto src_dims = src.dims();
+      if (src.ndim() == 0) {
+        VLOG(1) << "Copy output: index " << i << " skipped.";
+        Blob* dst = OperatorBase::OutputBlob(i);
+        dst->Reset(new Tensor<CPUContext>());
```

Comment:
How does the downstream op know whether its input tensor is CPU or IDEEP?

Reply:
@yinghai Thanks for raising this concern. Yes, this is still a workaround assuming the following op can only accept CPU tensors since IDEEP tensor doesn't support 0-dim. We are working on a complete solution to support 0-dim tensors and then such a workaround is not needed any longer.